packages/api/tests/lambdas/test-index-from-database.js

```diff
@@ -307,7 +291,6 @@ test.serial('Lambda successfully indexes records of all types', async (t) => {
     searchEs('pdr', esAlias, '20'),
```

Comment:
No to punt this but... https://bugs.earthdata.nasa.gov/browse/CUMULUS-3847 (punt)

Once we get there the work becomes deleting the test file wholesale. I think it's good to remove as we go (above conversation redux) but if we miss something 🤷. Probably best to handle the cleanup in a separate task in case there are issues.